RELEASE

```diff
@@ -0,0 +1 @@
+0.18.0
```

Comment:
I think there is something I'm not getting, but if RELEASE is the next intended version shouldn't this file be 1.0.0 then or?

Reply:
You are absolutely correct :)